inventory/host_vars/staging.coopcircuits.fr/config.yml

```diff
@@ -14,3 +14,6 @@ users_sysadmin:
   - "{{ core_devs }}"
   - paco
   - filipe
+
+# Images settings
+attachment_path: "home/openfoodnetwork/apps/openfoodnetwork/current/public/spree/products/:id/:style/:basename.:extension"
```

Comment:
this is the config in fr-prod, do we really need to use the same custom config in staging? I think we can use the default and configure the S3 FR accordingly.